hat/app/org/hatdex/hat/resourceManagement/HatServerProvider.scala

```diff
@@ -82,39 +82,40 @@ class HatServerProviderImpl @Inject() (
     configuration.get[FiniteDuration]("resourceManagement.serverIdleTimeout")
 
   def retrieve(hatAddress: String): Future[Option[HatServer]] = {
+    logger.warn(s"HatServiceProvider.retrieve: looking up hatAddress: server:${hatAddress}")
```

Comment:
is this the correct log level for this ? I know it could be important for future debugging but its a slippery slope .. maybe adding a comment to explain why it is warn not debug (or trace) ? or maybe change to info with a comment ?

Reply:
I noticed after as well, switching to .info